tsconfig.json

```diff
@@ -3,21 +3,16 @@
     "esModuleInterop": true,
     "skipLibCheck": true,
     "target": "esnext",
-    "allowJs": true,
     "resolveJsonModule": true,
-    "moduleDetection": "force",
     "moduleResolution": "bundler",
-    "module": "preserve",
+    "module": "es2015",
```

Comment:
@ndelangen I'm wondering if es2015 is actually supported or not. I think we could've kept `preserve` here, that's what we have in the addon-kit.